distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/AbstractIntegrationTest.java

```diff
@@ -672,200 +539,66 @@ protected Option installStartupFile(InputStream resourceInputStream, String dest
+    public static String getFileContent(String filePath) {
+        return getFileContent(filePath, ImmutableMap.<String, String>builder().build());
     }
 
     /**
-     * @deprecated since 2.8.0. Use {@link ServiceManager#waitForAllBundles()
-     * getServiceManager.waitForAllBundles()} instead.
+     * Variables to be replaced in a resource file should be in the format: $variableName$
+     * The variable to replace in the file should also also match the parameter names of the method calling getFileContent.
+     *
+     * @param filePath
+     * @param params
+     * @return
      */
-    @Deprecated
-    protected void waitForAllBundles() throws InterruptedException {
-        serviceManager.waitForAllBundles();
-    }
-
-    public class OpenSearchSourceProperties extends HashMap<String, Object> {
+    public static String getFileContent(String filePath, ImmutableMap<String, String> params) {
 
-        public static final String SYMBOLIC_NAME = "catalog-opensearch-source";
+        StrSubstitutor strSubstitutor = new StrSubstitutor(params);
 
-        public static final String FACTORY_PID = "OpenSearchSource";
+        strSubstitutor.setVariablePrefix(RESOURCE_VARIABLE_DELIMETER);
+        strSubstitutor.setVariableSuffix(RESOURCE_VARIABLE_DELIMETER);
+        String fileContent;
 
-        public OpenSearchSourceProperties(String sourceId) {
-            this.putAll(getMetatypeDefaults(SYMBOLIC_NAME, FACTORY_PID));
-
-            this.put("shortname", sourceId);
-            this.put("endpointUrl", OPENSEARCH_PATH.getUrl());
-        }
-
-    }
-
-    public class CswSourceProperties extends HashMap<String, Object> {
-
-        public static final String SYMBOLIC_NAME = "spatial-csw-source";
-
-        public static final String FACTORY_PID = "Csw_Federated_Source";
-
-        public static final String GMD_FACTORY_PID = "Gmd_Csw_Federated_Source";
-
-        public CswSourceProperties(String sourceId) {
-            this(sourceId, FACTORY_PID);
-        }
-
-        public CswSourceProperties(String sourceId, String factoryPid) {
-            this.putAll(getMetatypeDefaults(SYMBOLIC_NAME, factoryPid));
-
-            this.put("id", sourceId);
-            this.put("cswUrl", CSW_PATH.getUrl());
-            this.put("pollInterval", 1);
-        }
-
-    }
-
-    public class CswConnectedSourceProperties extends HashMap<String, Object> {
-
-        public static final String SYMBOLIC_NAME = "spatial-csw-source";
-
-        public static final String FACTORY_PID = "Csw_Connected_Source";
-
-        public CswConnectedSourceProperties(String sourceId) {
-            this.putAll(getMetatypeDefaults(SYMBOLIC_NAME, FACTORY_PID));
-
-            this.put("id", sourceId);
-            this.put("cswUrl", CSW_PATH.getUrl());
-            this.put("pollInterval", 1);
+        try {
+            fileContent =
+                    org.apache.commons.io.IOUtils.toString(AbstractIntegrationTest.class.getClassLoader()
```

Comment:
Same comment here about other classes using the AbstractIntegrationTest's class loader to get resources.